### DISCUSSION TITLE
Migrating to `@thulite` scope locks up Hugo (Recursive `public/`)

### DISCUSSION THREAD

**Discussion by l-zeuch** [gethyas/hyas#495] [Q&A] [answered]
Recently, I attempted to migrate our `package.json` from the old `@hyas` scope to the new `@thulite` scope, as per a relatively recent announcement, with the appropriate changes in our site's configuration. We're using the Doks theme.

However, when trying to build the site, it seems that hugo never finishes building. Inspecting the output directory (`public/`) it now contains a copy of the source directory and itself, repeat as often as you'd like. It eventually stops (when you cancel the build process), but you could theoretically let it fill up your drive if you so desire.

Changing everything back to the old `@hyas` scope (with the old versions that are in it), this issue does not arise. I'm not at all familiar with the ecosystem here, so I have exactly zero idea where the actual issue comes from, but it must be in a recent version of some doks-related package.

**Comment by h-enk** [comment, marked as answer]
It looks like something's off with your [configuration](https://getdoks.org/upgrade-guides/upgrade-to-v1/#5-configure-doks).

Can you share your repo?

**Reply by l-zeuch** [reply]
Thanks for the reply! Maybe? I initially thought it'd just be a simple search-and-replace, but if you could take a closer look (perhaps I missed something) that'd be greatly appreciated.

The repo in question is located at https://github.com/botlabs-gg/yagpdb-docs-v2.

My attempt at migrating to the `@thulite` namespace is on a separate branch: https://github.com/botlabs-gg/yagpdb-docs-v2/tree/migrate-thulite-namespace.

**Reply by h-enk** [reply]
You were almost there 😉 — see https://github.com/botlabs-gg/yagpdb-docs-v2/pull/52

Setting `baseurl = "http://localhost/"` in `config/_default/hugo.toml` is highly recommended. And, adding a `config/production/hugo.toml` to set `baseurl` to your production URL

**Reply by l-zeuch** [reply]
Thanks for your help! :pray: 

Yeah, we had some issues at the beginning with setting baseurl to *anything*, so we decided to go with the approach of setting it in a workflow at build time--I will push a separate to change these values around as you described.
